src/acachecontrol/acachecontrol.py

```diff
@@ -14,32 +14,32 @@ def __init__(
         self.cache = cache
         self._async_client_session = aiohttp.ClientSession()
 
-    def request(self, method, url, **params):
+    def _request(self, method, url, **params):
```

Comment:
What do you think if we will have methods "request" and "_request" as it in aiohttp?

Reply:
What's the reason?

Reply:
To support method request :)
Of course, somebody may use _request for that purpose, maybe it's not important for now